tests/test_class_structure.py

```diff
@@ -50,7 +50,7 @@ def test_subclasses():
     sc1 = [x[0] for x in subclasses1]
     sc2 = [x[0] for x in subclasses2]
     diff = set(sc1).difference(set(sc2))
-
+    print(str(diff))
```

Comment:
Can we remove this print statement now that debugging is done?

Reply:
Yep!